readme.md

```diff
@@ -34,24 +35,29 @@ gulp.task('default', () =>
 
 #### options
 
-XO [options](https://github.com/sindresorhus/xo#config) can be specified in package.json.
+##### quiet
 
-In the gulpfile you can specify the following options:
+Type: `boolean`<br>
+Default: `false`
 
-##### reporter
+Report errors only.
 
-Type: `string`<br>
-Default: [`eslint-formatter-pretty`](https://github.com/sindresorhus/eslint-formatter-pretty)
+### [xo.result(action)](https://github.com/adametry/gulp-eslint/#eslintresultaction)
 
-Any [ESLint reporter](http://eslint.org/docs/user-guide/command-line-interface#f---format).
+### [xo.results(action)](https://github.com/adametry/gulp-eslint/#eslintresultsaction)
 
-##### quiet
+### [xo.failOnError()](https://github.com/adametry/gulp-eslint/#eslintfailonerror)
 
-Type: `boolean`<br>
-Default: `false`
+### [xo.failAfterError()](https://github.com/adametry/gulp-eslint/#eslintfailaftererror)
 
-Report errors only.
+### [xo.format(formatter, output)](https://github.com/adametry/gulp-eslint/#eslintformatformatter-output)
+
+### [xo.formatEach(formatter, output)](https://github.com/adametry/gulp-eslint/#eslintformateachformatter-output)
```

Comment:
These should be sorted by usefulness:

- format
- failAfterError
- failOnError
- formatEach
- result
- results